README.md

```diff
@@ -25,6 +25,9 @@ Contributions are very welcome - please follow the [guidelines](CONTRIBUTING.md)
 - [nCov-Group Data Repository](https://2019-ncovgroup.github.io/data/#dataset-downloads): SMILES, fingerprints, descriptors, and images of millions of compounds.
 - [zinc20](https://files.docking.org/zinc20-ML/): ZINC20 library prepared for Deep Docking-accelerated virtual screening
 - [zinc22](https://cartblanche22.docking.org/): commercially-available compounds for virtual screening
+- [COCONUT](https://coconut.naturalproducts.net/): is an open source project for Natural Products (NPs) storage, search and analysis.
+- [nmrshiftdb2](https://nmrshiftdb.nmr.uni-koeln.de/nmrshiftdb/media-type/html/user/anon/page/default.psml/js_pane/P-Help;jsessionid=8DC5FC88F23A161E2276E00C85C184CA): is a NMR database (web database) for organic structures and their nuclear magnetic resonance (nmr) spectra. 
```

Comment:
```suggestion
- [nmrshiftdb2](https://nmrshiftdb.nmr.uni-koeln.de/A): is database for organic structures and their nuclear magnetic resonance (NMR) spectra. 
```

Reply:
and I guess the raw data is here https://sourceforge.net/projects/nmrshiftdb2/files/data/. Maybe this is also worth linking?

Reply:
some A letter in the last make link doesn't work. Try to edit link to be like this: https://nmrshiftdb.nmr.uni-koeln.de/nmrshiftdb
While some individuals struggle with Sourceforge, enter raw data on the final line([with raw data here](https://sourceforge.net/projects/nmrshiftdb2/files/data/)).

Reply:
>some A letter in the last make link doesn't work. Try to edit link to be like this: [nmrshiftdb.nmr.uni-koeln.de/nmrshiftdb](https://nmrshiftdb.nmr.uni-koeln.de/nmrshiftdb)

thanks for spotting!